pkg/serverless/trigger/events/events.go

```diff
@@ -230,7 +230,10 @@ type KinesisRecord struct {
 type EventBridgeEvent struct {
 	DetailType string `json:"detail-type"`
 	Source     string
-	StartTime  string
+	Time       time.Time
+	Detail     struct {
+		TraceContext map[string]string `json:"_datadog"`
+	}
```

Comment:
This allows us to easily unmarshall the trace context while avoiding other fields in `detail` that the agent doesn't care about (i.e. the user's custom fields)